src/lib/crypto/include/sol-message-digest.h

```diff
@@ -218,8 +230,8 @@ void sol_message_digest_del(struct sol_message_digest *handle);
  *        @c config->on_digest_ready() is called with the resulting digest
  *        as a blob.
  *
- * @return 0 on success, -errno otherwise. If error, then the input reference
- *         is not taken.
+ * @return 0 on success, @c -ENOSPC if sol_message_digest_config::tx_size is not zero and there's
+ * no more space left or -errno on error. If error or -ENOPSC, If error, then the input reference is not taken.
```

Comment:
I did not mention because the docs are not part of this PR. I don't want to break the docs generation